NAVAND-1380 platform workaround (#7245)
mapbox/mapbox-navigation-android: changelog/unreleased/bugfixes/7245.md

> - Added experimental and temporary(i.e. it will be removed in one of the next releases) `OnlineRouteAlternativesSwitch` which is automatically switching from the current offline route to an online route. It's designed for the case when platform's reachability API doesn't work reliably.

Reviewer: ```suggestion
- Added experimental and temporary (i.e. it will be removed in one of the next releases) `OnlineRouteAlternativesSwitch` which requests online route when the current route is offline and automatically switches if such a route is found. It's designed for the case when platform's reachability API doesn't work reliably.
```
